doc/design/mount-config-design.md

```diff
@@ -84,7 +84,7 @@ The following new command line argument will be added:
 
 | Name | Value |
 | -- | -- |
-| source | This is the source location of the mount in the insecure host. |
+| source | This is the source location of the mount, it has to be an absolute path. 1) For mounting hostfs, this path should exist on insecure host. 2) For mounting ext2, this path should point to a EXT2 archive on insecure host. 3) For mounting ramfs, this path should point to a CPIO archive within TEE filesystem(rootfs).  |
```

Comment:
I think I would replace "insecure host" with "the untrusted host" (since we general refer to trusted and untrusted entities).